visual permissions fixes

alxnddr opened the pull request:
1) Closes https://github.com/metabase/metabase/issues/21765

Open the [data permissions page](http://localhost:3000/admin/permissions/data/group) and ensure it handles short and long names correctly.
**Short name**
<img width="591" alt="Screenshot 2022-04-20 at 17 27 38" src="https://user-images.githubusercontent.com/14301985/164241322-120b2fce-bef8-45dc-a5fe-7776820fe0e8.png">

**Long name**
<img width="553" alt="Screenshot 2022-04-20 at 17 27 21" src="https://user-images.githubusercontent.com/14301985/164241387-57bfb763-26c4-4f9f-af8d-1c182debeacf.png">


2) Closes https://github.com/metabase/metabase/issues/21763
Open the [data model page](http://localhost:3000/admin/datamodel/database/5/table/25) and click on the original schema. Ensure the table name is not duplocated.

<img width="572" alt="Screenshot 2022-04-20 at 17 32 05" src="https://user-images.githubusercontent.com/14301985/164241886-664265eb-f1a0-4193-8535-035edc68f770.png">

alxnddr committed:
visual permissions fixes

iethree commented on frontend/src/metabase/admin/permissions/components/PermissionsTable/PermissionsTable.jsx:
```diff
@@ -102,10 +103,12 @@ export function PermissionsTable({
                     </EntityNameLink>
                   ) : (
-                    <EntityName>{entity.name}</EntityName>
+                    <EntityName>
+                      <Ellipsified>{entity.name}</Ellipsified>
+                    </EntityName>
                   )}
 
                   {entity.hint && (
```
this causes entity hints to get wrapped to the next line. I think they're meant to be on the same line

![Screenshot from 2022-04-20 16-30-05](https://user-images.githubusercontent.com/30528226/164335491-9eae3777-e501-4426-a811-4a083d23b5b0.png)

iethree commented:
maybe we should give the group name list on `/admin/people/groups` the `<Ellipsified>` treatment as well, or at least not wrap long names under the letter-icon.
![Screenshot from 2022-04-20 16-30-56](https://user-images.githubusercontent.com/30528226/164335728-a1b7ee23-0bb2-4594-b86c-eb1714e24ea7.png)

alxnddr committed:
fix permission table text wrap with an icon

alxnddr commented:
@iethree good catch, thanks! Fixed both

<img width="777" alt="Screenshot 2022-04-22 at 19 55 06" src="https://user-images.githubusercontent.com/14301985/164755362-8de25cf1-ffde-424a-b2c4-88ccaf3b492f.png">

On the groups page I added text wrapping instead of trimming similarly to the people page:
<img width="967" alt="Screenshot 2022-04-22 at 20 25 24" src="https://user-images.githubusercontent.com/14301985/164755568-7600cdfd-57ee-4b2f-864f-7c532eb3d81b.png">

iethree reviewed (approved):
:sparkling_heart:
